.devcontainer/images/Dockerfile

```
@@ -177,14 +177,19 @@ RUN NODE_ARCH=$([ "$TARGETARCH" = "arm64" ] && echo "arm64" || echo "x64") && \
 # =============================================================================
 # User Tools (status-line, xdg-open)
 # =============================================================================
-RUN ARCH=$([ "$TARGETARCH" = "arm64" ] && echo "arm64" || echo "amd64") && \
+RUN --mount=type=secret,id=github_token \
```

Hadolint (2.15.1)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)